src/CommonITILValidation.php

```diff
@@ -969,201 +969,162 @@ public function showSummary(CommonDBTM $item)
-                    $target_name = getUserName($row["items_id_target"]);
-                } elseif (is_a($row["itemtype_target"], CommonDBTM::class, true)) {
-                    $target = new $row["itemtype_target"]();
-                    $type_name = $target->getTypeName();
-                    if ($target->getFromDB($row["items_id_target"])) {
-                        $target_name = $target->getName();
-                    }
-                }
-                echo "<td>" . htmlspecialchars($type_name) . "</td>";
-                echo "<td>" . htmlspecialchars($target_name) . "</td>";
-                $is_answered = $row['status'] !== self::WAITING && $row['users_id_validate'] > 0;
-                echo "<td>" . ($is_answered ? htmlspecialchars(getUserName($row["users_id_validate"])) : '') . "</td>";
-                $comment_validation = RichText::getEnhancedHtml($this->fields['comment_validation'] ?? '', ['images_gallery' => true]);
-                echo "<td><div class='rich_text_container'>" . $comment_validation . "</div></td>";
-
-                $doc_item = new Document_Item();
-                $docs = $doc_item->find(["itemtype"          => $this->getType(),
-                    "items_id"           => $this->getID(),
-                    "timeline_position"  => ['>', CommonITILObject::NO_TIMELINE]
-                ]);
-                 $out = "";
-                foreach ($docs as $docs_values) {
-                     $doc = new Document();
-                     $doc->getFromDB($docs_values['documents_id']);
-                     $out  .= "<a ";
-                     $out .= "href=\"" . Document::getFormURLWithID($docs_values['documents_id']) . "\">";
-                     $out .= htmlspecialchars($doc->getField('name')) . "</a><br>";
-                }
-                 echo "<td>" . $out . "</td>";
+            $script = "";
+            if ($canedit) {
+                $params = ['type'             => $this->getType(),
+                    'parenttype'       => static::$itemtype,
+                    static::$items_id  => $this->fields[static::$items_id],
+                    'id'               => $row["id"]
+                ];
 
-                 echo "</tr>";
+                $editTitle = htmlspecialchars(__('Edit'), ENT_QUOTES, 'UTF-8');
+                $itemId = htmlspecialchars($item->fields['id'], ENT_QUOTES, 'UTF-8');
+                $rowId = htmlspecialchars($row["id"], ENT_QUOTES, 'UTF-8');
+                $rand = htmlspecialchars($rand, ENT_QUOTES, 'UTF-8');
+                $viewValidationId = htmlspecialchars($this->fields[static::$items_id], ENT_QUOTES, 'UTF-8');
+                $rootDoc = htmlspecialchars($CFG_GLPI["root_doc"], ENT_QUOTES, 'UTF-8');
+                $paramsJson = json_encode($params, JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_AMP | JSON_HEX_QUOT);
+
+                $script = <<<HTML
+                <span class='far fa-edit' style='cursor:pointer' title='$editTitle' 
```

Comment:
Other lists just trigger the edit by clicking anywhere in the row.
See `Rule::showActionsList` method.